guacamole_cloud_init.tf

```diff
@@ -5,31 +5,38 @@ data "template_cloudinit_config" "guacamole_cloud_init_tasks" {
   base64_encode = true
 
   part {
-    filename     = "install-certificates.yml"
-    content_type = "text/cloud-config"
+    content_type = "text/x-shellscript"
     content = templatefile(
-      "${path.module}/install-certificates.tpl.yml", {
+      "${path.module}/cloud-init/install-certificates.py", {
         cert_bucket_name   = var.cert_bucket_name
         cert_read_role_arn = var.guacamole_cert_read_role_arn
         server_fqdn        = var.guacamole_fqdn
     })
   }
 
   part {
-    filename     = "setup-guac-connection-sql.yml"
+    filename     = "write-guac-connection-sql-template.yml"
     content_type = "text/cloud-config"
     content = templatefile(
-      "${path.module}/setup-guac-connection-sql.tpl.yml", {
+      "${path.module}/cloud-init/write-guac-connection-sql-template.tpl.yml", {
+        guac_gophish_connection_name = var.guac_gophish_connection_name
+        pca_private_domain           = local.pca_private_domain
+        sql_template_fullpath        = "/root/guacamole_connection_template.sql"
+    })
+  }
+
+  part {
+    content_type = "text/x-shellscript"
+    content = templatefile(
+      "${path.module}/cloud-init/render-guac-connection-template-sql.py", {
```

Comment:
I would like some consistency between the naming for `write-guac-connection-sql-template.yml` and `render-guac-connection-template-sql.py` if possible.

Reply:
Good catch - fixed in https://github.com/cisagov/pca-terraform/pull/5/commits/c8573cb7a017867a4365716cb1c31cc5da18c508.